robottelo/host_info.py

```diff
@@ -1,15 +1,14 @@
 """Module that gather several informations about host"""
+import functools
```

Comment:
Why not `from functools import lru_cache`?

I mean - theoretical performance impact is probably negligible in context of whole project, but I wonder why you decided to import entire module instead of just part that you need.

Reply:
IDK I just thought it looked cleaner and more explicit that way. IMHO we do a little to much from x import y or *.

Reply:
Explicit is better than Implicit. Basically, it allows the reader to better understand what is the scope of lru_cache without always having to reference the import.
Also, python loads the entire functools module even if importing lru_cache, so there is no impact to performance. The only change would be the namespace of functools.lru_cache vs lru_cache